content/docs/languages/rust/40-supply-chain-analysis.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Supply chain analysis"
+slug: rust-supply-chain-analysis
+summary: "This section describes tricks for Rust unit testing"
+weight: 40
+---
+
+# Supply chain analysis
+
+## Vetting
+
+The tools in this section are more for "understanding" than "checking." E.g., running them does not produce "bug reports", but can help you assess maturity and security of dependencies. Tools below are rather "quantitative" than "qualitative" - you will need to do manuall, in-depth review of the outputs to extract any solid evidences about the maturity.
```

Comment:
```suggestion
The tools in this section are more for "understanding" than "checking." E.g., running them does not produce "bug reports", but can help you assess the maturity and security of dependencies. The tools below are more "quantitative" than "qualitative" - you will need to do manual, in-depth review of the outputs to extract any solid evidences about the maturity.
```
I'm not sure 'The tools below are more "quantitative" than "qualitative"' expresses whan you mean. You might just drop that part of the sentence.